Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM postgres:9.6-alpine
```

Comment:
I know we looked to swap over to Ubuntu, we can either make that change here or as a follow up. Just trying to break up my branch into more independent changes.

Reply:
Moved this with b409583 — the regression tests weren't working correctly with the Alpine image anyway (it has a different `diff` implementation) and I want to get those working before applying some of the other changes from my other branch.